app/components/Nav.tsx

```diff
@@ -1,5 +1,5 @@
 'use client';
-import { useState, useEffect } from 'react';
+import {useState, useEffect} from 'react';
 import Image from 'next/image';
 import logo from 'E:/intern-ecommerce-project/public/logo.png';
```

Comment:
Do not use the complete path name. You can use './logo.png'; it will work fine and is the correct way to reference the image path.